components/widgets/menu/menu_items/submenu_item/submenu_item.test.tsx

```diff
@@ -2,43 +2,52 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import {Provider} from 'react-redux';
+import configureStore from 'redux-mock-store';
 import {mount} from 'enzyme';
 
-import SubMenuItem from './submenu_item';
+import SubMenuItem from './index';
```

Comment:
we can still just test `./submenu_item.`. This way you can avoid `<Provider store={store}>` below. index file has only connected component so, we don't need to test that

Reply:
I just tried this but causes an error in the tests that have `subMenu` because `./submenu_item` uses the redux version in `./index`.